References.php

```diff
@@ -222,4 +222,29 @@ public function interlace(string $interlaceType): string
             'http://www.imagemagick.org/script/command-line-options.php#interlace'
         ));
     }
+
+    /**
+     * Checks that threshold value is valid according to ImageMagick command line reference.
+     */
+    public function threshold(string $threshold): float
+    {
+        $threshold = \trim($threshold);
+
+        if (substr($threshold, -1) == '%') {
+            $percent_num = substr($threshold, 0, -1);
+            if (is_numeric($percent_num))
+                return $threshold;
```

Comment:
Some code style change:
```suggestion
            if (is_numeric($percent_num)) {
                return $threshold;
            }
```